packages/storybook/src/nimble/button/button.mdx

```diff
@@ -1,45 +1,269 @@
+import { NimbleButton } from '@ni/nimble-react/dist/esm/button';
+import { NimbleSelect } from '@ni/nimble-react/dist/esm/select';
+import { NimbleListOption } from '@ni/nimble-react/dist/esm/list-option';
 
 <Meta of={buttonStories} />
 <Title of={buttonStories} />
 
-Per [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/button/) - A button is a
-widget that enables users to trigger an action or event, such as submitting a
-form, opening a dialog, canceling an action, or performing a delete operation.
+## Overview
 
-Consider <Tag name={anchorButtonTag} /> if you want a button that triggers
-navigation to a URL.
+A button is a widget that enables users to trigger an action or event, such as
+submitting a form, opening a dialog, canceling an action, or performing a delete
+operation. _Source: [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/button/)_
 
-<Canvas of={buttonStories.outlineButton} />
+<TableOfContents
+    title="On this page"
+    headingSelector="h2"
+    exclude={['Overview']}
+/>
+
+<img
+    alt="Button anatomy"
+    src={ButtonAnatomy}
+    style={{ maxWidth: '936px', width: '100%', display: 'block' }}
+/>
 
-## API
+## Implementation & API
 
 <Controls of={buttonStories.outlineButton} />
 
 <ComponentApisLink />
 
-## Styling
+## Usage
+
+### When to Use
+
+- Primary Actions: Use buttons for primary actions that users need to take, such
+  as submitting a form, saving changes, or proceeding to the next step.
+- Call to Action (CTA): Buttons are ideal for CTAs that encourage users to take
+  a specific action, like “Sign Up,” “Buy Now,” or “Learn More.”
+
+## Appearance & Styling
+
+Standard styling options for the button control are found below, in order of
+usage — most to least. Consider the surrounding context before employing an
+appearance variant.
+
+### 1. Ghost Button
+
+<Canvas of={buttonStories.ghostButton} />
+
+#### Your Go-to Option
+
+Consider the Ghost button as the default. Perfect for tight spaces and managing
+visual density, it can produce UIs that feel clean, airy, and open. Pairs well
+with Outline or Block buttons. Ensure your button doesn't blend in with body
+text, tabs, or links.
+
+### 2. Outline Button
+
+<Canvas of={buttonStories.outlineButton} />
+
+#### Slightly More Visible
+
+The Outline button serves as the secondary style, ideal when the Ghost button
+just isn't enough. It retains some of the Ghost button’s positive attributes,
+like a clean and light presence, while appearing more visually explicit about
+its functionality.
+
+### 3. Block Button
+
+<Canvas of={buttonStories.blockButton} />
+
+#### The Stand Out
+
+The Block button is more visually prominent and functionally direct than the
+Outline or Ghost options. Use it when higher visibility is a priority or in busy
+environments where important controls might be missed. This appearance helps
+ensure critical actions aren’t overlooked.
+
+### Button Variants
+
+There are two available values for appearanceVariant — primary and accent —
```

Comment:
appearanceVariant should either use code font and the attribute-style name (`appearance-variant`) or plain English (appearance variant). Since the rest of this section is using English names (e.g. "Outline and Block appearances") I'd vote for the latter.